### PR TITLE
[MIG] [16.0] commission_formula

### PR DESCRIPTION
**Pull request by Nikul-OSI**

### CODE REVIEW

File: commission_formula/__manifest__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "summary": "Commissions computed by formulas",
     "author": "Abstract,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/commission",
-    "depends": ["commission"],
+    "depends": ["commission", "account"],
```
Comment (pedrobaeza) [inline commission_formula/__manifest__.py:13]: This added dependency doesn't seem correct.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The module version bumping is not in the correct commit.

File: commission_formula/__manifest__.py (hunk 1 above)
Comment (Nikul-OSI) [reply commission_formula/__manifest__.py:13]: Here we used move line https://github.com/OCA/commission/pull/409/commits/f141ae9381f3240cc33a477869006759870d5627#diff-628b4901eac43e5fc9b48afa3b145f4d37d09a8ce33340b551f2297524fbc072R11

Comment (pedrobaeza) [reply commission_formula/__manifest__.py:13]: The link is not correctly shown, but I have checked full code and I don't see such dependency.

**Pull request closed without merging**